anchor/anchor_parser.go

```diff
@@ -45,11 +46,12 @@ func ParseAnchors(protoAnchors []*yotiprotoattr.Anchor) []*Anchor {
 					continue
 				}
 				anchorType = parsedAnchorType
-				extensions = append(extensions, value)
+				extension = value
+				break certs
```

Comment:
We should... probably refactor this goto out, using an unexported function that returns.

Reply:
Fair point, I've updated in https://github.com/getyoti/yoti-go-sdk/pull/99/commits/226373528a1c1a69f018ffc6ead9eff60e9b5ba1